src/Microsoft.AspNet.SignalR.Client.UWP/Microsoft.AspNet.SignalR.Client.UWP.csproj

```diff
@@ -336,6 +333,14 @@
     <EmbeddedResource Include="Resources.resx" />
     <EmbeddedResource Include="Store.Resources.resx" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
```

Comment:
Why?

Reply:
Ditto above. The project was upgraded when I opened it in VS 2017. I felt like it was good to just let it happen rather than forcing people to use VS 2015 to open the solution.